### PR TITLE
Devicetree release notes for v3.2

### PR DESCRIPTION
**Pull request by mbolivar-nordic**
And a sneaker patch for a bindings problem I found along the way.

### CODE REVIEW

**Review by dcpleung — DISMISSED** [review]

**Review by erwango — DISMISSED** [review]
Approved & Thanks for the fix.

**Review by gmarull — DISMISSED** [review]
Best release notes so far

**Review by henrikbrixandersen — DISMISSED** [review]

File: doc/releases/release-notes-3.2.rst [hunk 1]
```diff
@@ -462,10 +462,454 @@ Build System
...
+     if (!device_is_ready(my_dev)) {
+             /* device exists, but it failed to initialize */
+     } else {
+             /* device is ready for use */
+     }
+
+  As shown in the above snippet, :c:macro:`DEVICE_DT_GET` should generally be
+  used instead of ``device_get_binding()`` when getting device structures from
+  devicetree nodes. Note that you can pass ``DEVICE_DT_GET`` any devicetree
+  :ref:`node identifier <dt-node-identifier>` -- you don't have to use
```
Comment (stephanosio) [inline doc/releases/release-notes-3.2.rst:531]:
```suggestion
  :ref:`node identifier <dt-node-identifiers>` -- you don't have to use
```

**Review by stephanosio — DISMISSED** [review]

File: doc/releases/release-notes-3.2.rst [hunk 2]
```diff
@@ -462,10 +462,454 @@ Build System
...
+      * :dtcompatible:`nxp,lpc-iocon-pinctrl`: new ``nxp,analog-alt-mode`` property
+      * removed deprecated ``nxp,lpc-iap`` binding
+      * :dtcompatible:`nxp,imx-csi`: new ``sensor`` property replacing the
+        ``sensor-label`` property
+      * :dtcompatible:`nxp,imx-lpi2c`: new ``scl-gpios``, ``sda-gpios`` properties
+
+    * STM32 SoCs:
+
+      * :dtcompatible:`st,stm32-adc`: new ``has-vbat-channel`` property
+      * :dtcompatible:`st,stm32-can`: new ``ts-cal-resolution`` property;
```
Comment (henrikbrixandersen) [inline doc/releases/release-notes-3.2.rst:848]: Where do you see this new `ts-cal-resolution` property?

**Review by henrikbrixandersen — CHANGES_REQUESTED** [review]

File: doc/releases/release-notes-3.2.rst (hunk 2 above)
Comment (erwango) [reply doc/releases/release-notes-3.2.rst:848]: Indeed, this should be `st,stm32-temp-cal.yaml`

**Comment by mmahadevan108** [comment]
@mbolivar-nordic, can you help take a look at the comments.

**Review by MaureenHelm — DISMISSED** [review]

File: doc/releases/release-notes-3.2.rst (hunk 2 above)
Comment (mbolivar-nordic) [reply doc/releases/release-notes-3.2.rst]: I guess I just made it up. Thanks @henrikbrixandersen ; I removed that.

**Review by henrikbrixandersen — APPROVED** [review]

**Review by stephanosio — APPROVED** [review]

**Review by galak — APPROVED** [review]

**Pull request merged**